bundler/lib/bundler/rubygems_integration.rb

```diff
@@ -48,7 +48,13 @@ def mark_loaded(spec)
     end
 
     def validate(spec)
-      Bundler.ui.silence { spec.validate(false) }
+      Bundler.ui.silence do
+        if spec.respond_to?(:validate_for_resolution)
```

Comment:
Is this reasonable so new versions of Bundler still work with versions of RubyGems that don't have this method defined?